### ISSUE TITLE
How to generate mask images?

### ISSUE THREAD

**Issue by xind** [tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation#50]
I want to train with train2017 and I've tried using coco.annToMask() to get mask but the output are different from yours.
How can I get mask for train2017?

**Comment by tensorboy** [comment]
Training steps of the official repo: https://github.com/ZheC/Realtime_Multi-Person_Pose_Estimation will do that. :)

**Comment by xind** [comment]
Thanks, I will try.

**Comment by xind** [comment]
Another issue I found is why the mask images I generated with genCOCOMask.m are different from [yours](https://www.dropbox.com/s/bd9ty7b4fqd5ebf/mask.tar.gz?dl=0)?

Here's the comparison:
https://imgur.com/a/LBiC2OC

**Comment by tensorboy** [comment]
I'm not sure if the processing you generated mask is right, but that mask is for the unlabeled person, and the mask you made seems like the ground truth semantic label.

**Comment by xind** [comment]
I just ran the training steps from the official repo as you told:
- Run getANNO.m in matlab to convert the annotation format from json to mat in dataset/COCO/mat/.
- Run genCOCOMask.m in matlab to obatin the mask images for unlabeled person. You can use 'parfor' in matlab to speed up the code.

What changes should be made to generate mask images like yours?

**Comment by tensorboy** [comment]
Hi, @xind. I just generated the mask with the same processing as you as I can remember. I don't think there are many differences for the final result caused by the mask. :)

**Comment by qiuzhongwei-USTB** [comment]
> I want to train with train2017 and I've tried using coco.annToMask() to get mask but the output are different from yours.
> How can I get mask for train2017?

Hi, I want train coco2017 like you, could you be kind to share me the changes on this repo you have done?
